app/views/layouts/_top_nav.html.erb

```diff
@@ -13,23 +13,29 @@
       </button>
       <%= link_to :CampZone, director_dashboard_index_path, class:"navbar-brand" %>
     </div>
-    <div class="collapse navbar-collapse" id="user-dropdown">
+    <div class="collapse navbar-collapse" id="nav-dropdown">
       <ul class="nav navbar-nav pull-right">
         <% if current_user %>
-          <li class="navbar-text hidden-xs">Welcome, <%= current_user.first_name %></li>
           <% if current_user.role == "director" %>
             <li><%= link_to :Dashboard, director_dashboard_index_path %></li>
             <li><%= link_to :Counselors, director_counselors_path %></li>
             <li><%= link_to :Groups, director_groups_path %></li>
             <li><%= link_to :Kids, director_kids_path %></li>
           <% end %>
           <% if current_user.role == "counselor" %>
-            <li><%= link_to :Profile, counselor_profile_path %></li>
```

Comment:
Removed it because it's replaced with settings slightly down below. I am having trouble visualizing what the counselor profile page has that the settings doesn't. Thoughts?